batch/batch/worker_config.py

```diff
@@ -112,11 +114,22 @@ def resources(self, cpu_in_mcpu, memory_in_bytes):
                           'quantity': cpu_in_mcpu})
 
         resources.append({'name': f'memory/{self.instance_family}-{preemptible}/1',
-                          'quantity': memory_in_bytes / 1024 / 1024})
-
-        # the factors of 1024 cancel between GiB -> MiB and fraction_1024 -> fraction
-        resources.append({'name': f'boot-disk/{self.boot_disk_type}/1',
-                          'quantity': self.boot_disk_size_gb * worker_fraction_in_1024ths})
+                          'quantity': math.ceil(memory_in_bytes / 1024 / 1024)})
```

Comment:
memory_in_bytes should always be divisible by 1024 * 1024, right? Let's change this to:
```
assert memory_in_bytes % (1024 * 1024) == 0
...
    'quantity': memory_in_bytes // 1024 // 1024
```